docs/examples/pam.d/fty

```diff
@@ -36,7 +36,7 @@
 # The 42ity requirements for password:
 #   8 characters, including one digit and one 'other' character
 #   flag 'enforce_for_root' is for 'bios-passwd' tool, which runs under root
-    password	requisite	pam_cracklib.so enforce_for_root minlen=8 dcredit=-1 ocredit=-1 ucredit=0 lcredit=0
+    password	include		fty-password
```

Comment:
Did you test this change? A couple of lines below there is a `@include <filename>` syntax instead... I do not know OTOH which one is correct, but the existing one is there for a long time :)

Reply:
Yes, I have tested this change.
The both are correct but :
The `@include <filename>` will include all the content from the file.
`<type>   include  <file>` will include all lines of the given type from the file.

I use this one because it seems better supported and, unlike the common-* includes, this line can't be just commented.

But I can change for a better homogeneity.